homeassistant/helpers/event.py

```diff
@@ -139,7 +139,7 @@ def remove() -> None:
 def async_track_state_change(
     hass: HomeAssistant,
     entity_ids: str | Iterable[str],
-    action: Callable[[str, State, State], None],
+    action: Callable[[str, State, State], None | Awaitable],
```

Comment:
I think it's nicer when `| None` comes in the end.
```suggestion
    action: Callable[[str, State, State], Awaitable | None],
```